minecraft/src/main/java/net/fabricmc/loader/impl/game/minecraft/McLibrary.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.loader.impl.game.minecraft;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.loader.impl.game.LibClassifier.LibraryType;
+
+enum McLibrary implements LibraryType {
+	FABRIC_LOADER_MC(null, true, "net/fabricmc/loader/impl/game/minecraft/MinecraftGameProvider.class"),
+	MC_CLIENT(EnvType.CLIENT, "net/minecraft/client/main/Main.class", "net/minecraft/client/MinecraftApplet.class", "com/mojang/minecraft/MinecraftApplet.class"),
+	MC_SERVER(EnvType.SERVER, "net/minecraft/server/Main.class", "net/minecraft/server/MinecraftServer.class", "com/mojang/minecraft/server/MinecraftServer.class"),
+	MC_COMMON("net/minecraft/server/MinecraftServer.class"),
+	MC_BUNDLER(EnvType.SERVER, "net/minecraft/bundler/Main.class"),
+	REALMS(EnvType.CLIENT, "realmsVersion"),
+	MODLOADER("ModLoader"),
+	LOG4J_API("org/apache/logging/log4j/LogManager.class"),
+	LOG4J_CORE("META-INF/services/org.apache.logging.log4j.spi.Provider", "META-INF/log4j-provider.properties"),
+	LOG4J_CONFIG("log4j2.xml"),
+	LOG4J_PLUGIN("com/mojang/util/QueueLogAppender.class"),
```

Comment:
We also use `net/minecrell/terminalconsole/util/LoggerNamePatternSelector` in dev.